test/cosmoz-bottom-bar.test.js

```diff
@@ -0,0 +1,220 @@
+import {
+	assert, fixture, html
+} from '@open-wc/testing';
+
+import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer';
+
+import '../cosmoz-bottom-bar.js';
+import './helpers/fixed-size-item.js';
+import { Polymer } from '@polymer/polymer/polymer-legacy';
```

Comment:
Is it used?

Reply:
Ah that was sloppy of me! I'll try to fix it along with the demos during the day